### PR TITLE
Readds Shadowlings 2.0

### PR DESCRIPTION
**Pull request by Buggy123**
See #30053 for details.
:cl: 
The Revolution gamemode has been replaced with Shadowlings.
/:cl:

### CODE REVIEW

**Comment by awesine** [comment]
nobody is going to accept that revs is going to be removed. If you want shadowlings, you'll have a far greater time just adding shadowlings. But revs is a good part of /tg/ history and ss13 as its core, and it's not going to be removed any time soon, especially by literally who.

**Comment by awesine** [comment]
Also you didn't even do the format right.

**Comment by awesine** [comment]
Considering how quickly this was done too, i wouldn't be surprised if you just copypasted it from hippie or what not.

**Comment by Spaceinaba** [comment]
I hate revs with a passion but this is not the way to go about this

**Comment by Buggy123** [comment]
Nobody? Then why do people literally constantly ask for it to be removed every single time there is a rev round? I have yet to see a single "remove traitors" "remove bloodcult" "remove extended" or any such. I think the only ones i HAVE seen are "remove changling" and "remove clockcult" and those are MUCH rarer than "remove revs".

"remove revs" occurs in AT LEAST 50% of rev rounds if not more. Face it, a large portion of the server does not like revs.

**Comment by Buggy123** [comment]
Fuck it doesn't compile.

File: code/datums/mind.dm [hunk 1]
```diff
@@ -236,10 +230,18 @@
 	remove_nukeop()
 	remove_wizard()
 	remove_cultist()
-	remove_rev()
+	remove_shadowling()
```
Comment (RemieRichards) [inline code/datums/mind.dm]: shame this isn't a proc on the universe.

Comment (Buggy123) [reply code/datums/mind.dm]: kek

**Comment by awesine** [comment]
@Buggy123 
i ded is literally the only reason you have for removing revs. stop being salty. this won't get merged anyway, i'll bet you my life on it. if it doesn't get merged, it'll get reverted within a week due to the backlash.

revs is the best gamemode and that is fact.

**Comment by awesine** [comment]
@Buggy123 
remove extended happens every time there is an extended round. constantly people whine to nerf things they get killed by.

do they get nerfed? does extended get removed?

no

i ded

grow up

**Comment by Buggy123** [comment]
Do you want me to start screenshotting people saying "remove revs"

or maybe do a ctrl+f in the logs and compare the 94,881 results for "remove revs" to the 2 results for "remove traitors"?

**Comment by awesine** [comment]
anyway the format isn't done right so it'll get closed

**Comment by Buggy123** [comment]
What format?

**Comment by awesine** [comment]
@Buggy123 
#33885

**Comment by Buggy123** [comment]
Done.

**Comment by MoreRobustThanYou** [comment]
if you're gonna copy my code don't do it worse

**Comment by awesine** [comment]
the people saying "remove revs" are a small vocal group of people who need to grow up and get rid of "MUH ARRR PEEE" "MUH IMMERSHUNS" "MUH SLOW PACED ROUNDS" mentality


and yes, show me the "94,881" times people have said remove revs. Ninety four thousand, eight hundred and eighty one times. I want screenshots of each individual one.
99% chance it's going to be the same twenty people

grow up

revs is a good gamemode and if you can't realize that you shouldn't be playing ss13

**Comment by optimumtact** [comment]
this isn't even funny

**Comment by awesine** [comment]
i like you now @optimumtact

**Pull request closed without merging**